src/Simplify.cpp

```diff
@@ -3765,6 +3765,10 @@ Stmt simplify_exprs(Stmt s) {
     return SimplifyExprs().mutate(s);
 }
 
+bool can_prove(Expr e) {
```

Comment:
I'm a little worried about this being called for non-Bool expressions and some one expecting it to return true if the Expr is non-zero. Perhaps an assert that the type of the passed in Expr is Bool?